pcdsdevices/gasdet.py

```diff
@@ -0,0 +1,345 @@
+        EpicsSignal,
+        '{self.prefix}:{self.module}:MType',
+        kind='config',
+        string=True
+    )
+    acq_mode = FCpt(
+        EpicsSignal,
+        '{self.prefix}:{self.module}:MMode',
+        kind='config',
+        string=True
+    )
+    acq_mode_flags = FCpt(
+        EpicsSignal,
+        '{self.prefix}:{self.module}:MModeFlags',
+        kind='config',
+        string=True
+    )
+    clock_type = FCpt(
+        EpicsSignalRO,
+        '{self.prefix}:{self.module}:MClockType',
+        kind='config',
+        string=True
+    )
+    crate_number = FCpt(
+        EpicsSignalRO,
+        '{self.prefix}:{self.module}:MCrateNb',
+        kind='config'
+    )
+    nADC_bits = FCpt(
+        EpicsSignalRO,
+        '{self.prefix}:{self.module}:MNbrADCBits',
+        kind='config'
+    )
+    crate_slot = FCpt(
+        EpicsSignalRO,
+        '{self.prefix}:{self.module}:MPosInCrate',
+        kind='config'
+    )
+    temp = FCpt(
+        EpicsSignalRO,
+        '{self.prefix}:{self.module}:MTemperature_m',
+        kind='config'
+    )
+    input_freq = FCpt(
+        EpicsSignalRO,
+        '{self.prefix}:{self.module}:MInputFrequency',
+        kind='config'
+    )
+
+    def __init__(self,
+                 prefix,
+                 module,
+                 platform=None,
+                 name='acqiris_module',
+                 **kwargs):
+        self.module = module
+        self.prefix = prefix
+        self.platform = platform
+        if len(self.module) != 3:
+            raise ValueError("acqiris 'module'"
+                             + " expects a 3-digit module identifier,"
+                             + " e.g. '240'")
+        self._mod_pref = self.module[:2]
+        super().__init__(prefix, name=name, **kwargs)
+
+    def start(self):
+        """
+        Start the acqiris DAQ
+        """
+        self.run_state.put(0)
+
+    def stop(self):
+        """
+        Stop the acqiris DAQ
+        """
+        self.run_state.put(1)
+
+
+class gasdet(Device):
+    """
+    Class to read out pulse energy from a PMT/diode as digitized
+    by an acqiris in the GasDetDAQ EPICS IOC.
+
+    Parameters
+    ----------
+    prefix : ``str``
+        The PV base of the digitizer.
+    platform : ``str``
+        This a legacy PV segment found in the GasDetDAQ IOC.
+        For example, '202' applies to
+        the HXR gas detector system... for some reason.
+    module : ``str``
+        3-digit number representing the digitizer board, e.g. '240', '360'.
+    channel : ``str``
+        The digitizer channel.
+
+    """
+    mJ = Cpt(
+        EpicsSignalRO,
+        ':ENRC',
+        kind='hinted'
+    )
+    time = Cpt(
+        EpicsSignalRO,
+        ':TIME',
+        kind='hinted'
+    )
+    cal = Cpt(
+        EpicsSignalRO,
+        ':CALI',
+        kind='hinted'
+    )
+    offset = Cpt(
+        EpicsSignalRO,
+        ':OFFS',
+        kind='hinted'
+    )
+    BG_start_index = Cpt(
```

Comment:
Github auto-coloring these differently is a clue that you've broken a convention
```suggestion
    bg_start_index = Cpt(
```